app/javascript/components/Pages/IcebreakerQuestion.js

```diff
@@ -91,15 +90,17 @@ const IcebreakerQuestion = ({data, setData, saveDataToDb, steps, service}) => {
                 }
               </div>
             </div>
-            <div className='d-flex justify-content-between m-3'>
-              <ShoutOutIcon/>
-              <BtnBack addClass="answer-custom" onClick={backHandling}/>
-              <BtnPrimary addClass="answer-custom" onClick={handlingOnClickNext} text={isEmptyStr(funQuestionBody) ? 'Skip to Results' : 'Submit'} />
-              <HelpIcon/>
+            <div className='d-flex placement-buttons justify-content-between col-6 offset-3 pb-53 mt-5'>
```

Comment:
the same

Reply:
Thank you  I think this decision more preferable in this case.